hazelcast/src/main/java/com/hazelcast/jet/impl/JobExecutionService.java

```diff
@@ -218,7 +218,7 @@ public ClassLoader getProcessorClassLoader(long jobId, String name) {
     }
 
     private Map<String, ClassLoader> createProcessorClassLoaders(long jobId, JobConfig jobConfig) {
-        String customLibDir = nodeEngine.getConfig().getProperty(JetProperties.PROCESSOR_CUSTOM_LIB_DIR.getName());
+        String customLibDir = nodeEngine.getConfig().getProperty(ClusterProperty.PROCESSOR_CUSTOM_LIB_DIR.getName());
```

Comment:
This should actually be 
```suggestion
        String customLibDir = nodeEngine.getProperties().getString(ClusterProperty.PROCESSOR_CUSTOM_LIB_DIR);
```